core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestratorSpec.scala

```diff
@@ -52,13 +51,6 @@ class BillingProjectOrchestratorSpec extends AnyFlatSpec {
   )
 
   val billingProfileId: UUID = UUID.randomUUID()
-  val azureBillingProfile: ProfileModel = new ProfileModel()
-    .id(billingProfileId)
-    .tenantId(UUID.randomUUID())
-    .subscriptionId(UUID.randomUUID())
-    .cloudPlatform(bio.terra.profile.model.CloudPlatform.AZURE)
-    .managedResourceGroupId("fake-mrg")
-    .createdDate("2023-09-12T22:20:48.949Z")
```

Comment:
we had a bunch of unused test fixtures - which relied on the BPM models. Now deleted.